src/index.js

```diff
@@ -95,7 +94,12 @@ export const RCComponent = ({
   return (
     <ToastBarProvider>
       <RCInstanceProvider value={{ RCInstance }}>
-        <Box width={width}>
+        <Box
+          width={width}
+          overflowX={'hidden'}
+          overflowY={'hidden'}
+          maxHeight={'100vh'}
```

Comment:
what are the repercussions of doing this change?

Reply:
For the maxHeight: (100vh). I set it to this because when we go to full screen mode for the component,  we wouldn't want the input textbox to overflow beneath the window (causing the wierd vertical scrolling issue I mentioned in #144).  Hence, setting it to a max height of 100vh avoids this, and there's no change to the user experience when using the component.

For OverflowX and OverlowY: Since all the content is within the same window and there's no need to scroll any direction, I have set the overflow to hidden (no scrolling). 
Let me know if you'd like to change it back.
I have tested these UI changes in a responsive setting as shown below:

https://user-images.githubusercontent.com/80944218/223284851-61963fdc-c558-4d2e-8edd-02d4988cb938.mp4

Reply:
Amazing!